src/operator/tensor/init_op.h

```diff
@@ -250,12 +284,12 @@ inline bool InitShape(const nnvm::NodeAttrs& attrs,
   return shape_is_known(out_attrs->at(0));
 }
 
-template<typename ParamType>
+template<typename ParamType, int inum = 0U>
```

Comment:
nit: changing `inum` to `num_in` is more readable

Reply:
Changed~